### ISSUE TITLE
Feature Request for Subtensor Documentation

### ISSUE THREAD

**Issue by Jackalgirl** [opentensor/developer-docs#216]
In your copious spare time (joking!), would it be possible to include some additional information, such as how to get to the log files for the different kinds of node installation/starting options are, and what a normally-running node would look like?

So for example, if someone is running a lite node they've installed from source, how to look at the logs and see if it's still warping, or how to find the best block?

**Comment by rajkaramchedu** [comment]
Yes. Also see the issue on this topic for context: https://github.com/opentensor/developer-docs/issues/188
